src/pet.js

```diff
@@ -0,0 +1,69 @@
+const INITIAL_AGE = 0;
+const MINIMUM_HUNGER = 0;
+const MAX_FITNESS = 10;
+const MAX_AGE = 30;
+const MAX_HUNGER = 10;
+const MINIMUM_FITNESS = 0;
+
+function Pet(name) {
+  this.name = name;
+  this.age = INITIAL_AGE;
+  this.hunger = MINIMUM_HUNGER;
+  this.fitness = MAX_FITNESS;
+  this.children = [];
+}
+
+Pet.prototype = {
+  get isAlive() {
+    return (
+      this.age < MAX_AGE &&
+      this.hunger < MAX_HUNGER &&
+      this.fitness > MINIMUM_FITNESS
+    );
+  },
+};
+
+Pet.prototype.growUp = function () {
+  if (!this.isAlive) {
+    throw new Error("Your pet is no longer alive :(");
```

Comment:
This `Your pet is no longer alive :(` message is used several times in the file, could it be put into a constant along with the values at the top?